nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1960,7 +1961,10 @@ private Description matchDereference(
    * @return the error description
    */
   private Description createErrorDescription(
-      MessageTypes errorType, Tree errorLocTree, String message, TreePath path) {
+      com.uber.nullaway.ErrorMessage.MessageTypes errorType,
```

Comment:
This should be `ErrorMessage.MessageTypes`, no? Rather than the FQN.

Reply:
Done 👍